pkg/config/plugins/resources/postgres/config.go

```diff
@@ -50,7 +52,10 @@ type PostgresStoreConfig struct {
 	// Driver to use, one of: pgx, postgres
 	DriverName string `json:"driverName" envconfig:"kuma_store_postgres_driver_name"`
 	// Connection Timeout to the DB in seconds
+	// Deprecated: it's used for MaxIdleConnectionTime (applied only when driverName=pgx), use `MaxConnectionIdleTime` instead.
```

Comment:
isn't this connection timeout used in: https://github.com/kumahq/kuma/blob/master/pkg/config/plugins/resources/postgres/config.go#L105 which is used for default postgres driver?